spec/spec_helper.rb

```diff
@@ -1,24 +1,24 @@
-require 'bundler'
-
 ENV['RACK_ENV'] ||= 'test'
+
+require 'bundler'
 Bundler.require(:default, :test)
 
 require File.expand_path('../../config/environment', __FILE__)
 require 'rspec'
 require 'capybara/dsl'
 
 Capybara.app = LittleShopApp
-
-DatabaseCleaner.strategy = :truncation
-
-RSpec.configure do |c|
-  c.before(:all) do
-    DatabaseCleaner.clean
-  end
-  c.after(:each) do
-    DatabaseCleaner.clean
-  end
-end
+# 
+# DatabaseCleaner.strategy = :truncation
+#
+# RSpec.configure do |c|
+#   c.before(:all) do
+#     DatabaseCleaner.clean
+#   end
+#   c.after(:each) do
```

Comment:
@lilwillifo Is there a particular reason you used :all on before and :each on after?